### PR TITLE
More cvar descriptions and minor corrections.

### PR DESCRIPTION
**Pull request by KuehnhammerTobias**
# More CVAR descriptions and minor corrections.
This pull request is an extension to my previous pull request, addressing issue: https://github.com/ec-/Quake3e/issues/188

New CVAR descriptions:

cl_activeAction "Contents of this variable will be executed upon first frame of play. Note: It is cleared every time it is executed."
cl_lanForcePackets "Bypass \\cl_maxpackets for LAN games, send packets every frame."
sv_levelTimeReset "Whether or not to reset leveltime after new map loads."
joy_threshold "Threshold of joystick moving distance."
ttycon "Terminal tab-completion and history support."
ttycon_ansicolor, "Terminal display console color codes."

All these CVAR descriptions come from ETe, which **ensiform** pointed out to me.

**zturtleman** also helped me. He suggested more correct formulations for many explanations:

s_sdlBits "Bits per-sample to request for SDL audio output (possible options: 8 or 16). When set to 0 it uses 16."
s_sdlChannels "Number of audio channels to request for SDL audio output. The Quake 3 audio mixer only supports mono and stereo. Additional channels are silent."
s_sdlDevSamps "Number of audio samples to provide to the SDL audio output device. When set to 0 it picks a value based on s_sdlSpeed."
s_sdlMixSamps "Number of audio samples for Quake 3's audio mixer when using SDL audio output."
s_sdlCapture "Set to 1 to enable SDL audio capture."
r_nocurves "Set to 1 to disable drawing world bezier curves. Set to 0 to enable."
r_drawworld "Set to 0 to disable drawing the world. Set to 1 to enable."
r_vertexLight "Set to 1 to use vertex light instead of lightmaps."
r_portalOnly "Set to 1 to turn off stencil buffering for portals."

Thank you both for the support!

For any changes, if the descriptions are wrong or not working, please do not blame them, but me.

I also made some changes myself, mainly spelling mistakes that my translation program complained about were corrected.
This affects existing descriptions as well as the quake3e.htm file.

## ***What is changing:***
  + Add new descriptions for some CVARs with a missing description.
  + Corrected some existing descriptions.
  + Fixing a few spelling mistakes.
#

### CODE REVIEW

File: code/renderer/tr_init.c [hunk 1]
```diff
@@ -1481,7 +1481,7 @@ static void R_Register( void )
 	r_roundImagesDown = ri.Cvar_Get ("r_roundImagesDown", "1", CVAR_ARCHIVE_ND | CVAR_LATCH );
 	ri.Cvar_SetDescription( r_roundImagesDown, "When images are scaled, round images down instead of up." );
 	r_colorMipLevels = ri.Cvar_Get ("r_colorMipLevels", "0", CVAR_LATCH );
-	ri.Cvar_SetDescription( r_colorMipLevels, "Debugging tool to artificially color different mipmap levels so that they are more apparent (defaul 0)." );
+	ri.Cvar_SetDescription( r_colorMipLevels, "Debugging tool to artificially color different mipmap levels so that they are more apparent (default 0)." );
```
Comment (ec-) [inline code/renderer/tr_init.c:1484]: no need to print default value in description

File: code/renderer2/tr_init.c [hunk 2]
```diff
@@ -1122,15 +1122,15 @@ static void R_Register( void )
...
+	ri.Cvar_SetDescription( r_ext_max_anisotropy, "Sets maximum anisotropic level for your graphics driver. Requires \\r_ext_texture_filter_anisotropic." );
 
 	r_picmip = ri.Cvar_Get ("r_picmip", "0", CVAR_ARCHIVE | CVAR_LATCH );
 	ri.Cvar_CheckRange( r_picmip, "0", "16", CV_INTEGER );
 	ri.Cvar_SetDescription( r_picmip, "Set texture quality, lower is better." );
 	r_roundImagesDown = ri.Cvar_Get ("r_roundImagesDown", "1", CVAR_ARCHIVE | CVAR_LATCH );
 	ri.Cvar_SetDescription( r_roundImagesDown, "When images are scaled, round images down instead of up." );
 	r_colorMipLevels = ri.Cvar_Get ("r_colorMipLevels", "0", CVAR_LATCH );
-	ri.Cvar_SetDescription( r_colorMipLevels, "Debugging tool to artificially color different mipmap levels so that they are more apparent (defaul 0)." );
+	ri.Cvar_SetDescription( r_colorMipLevels, "Debugging tool to artificially color different mipmap levels so that they are more apparent (default 0)." );
```
Comment (ec-) [inline code/renderer2/tr_init.c:1133]: no need to print default value in description

File: code/renderervk/tr_init.c [hunk 3]
```diff
@@ -1526,7 +1526,7 @@ static void R_Register( void )
 	r_roundImagesDown = ri.Cvar_Get ("r_roundImagesDown", "1", CVAR_ARCHIVE_ND | CVAR_LATCH );
 	ri.Cvar_SetDescription( r_roundImagesDown, "When images are scaled, round images down instead of up." );
 	r_colorMipLevels = ri.Cvar_Get ("r_colorMipLevels", "0", CVAR_LATCH );
-	ri.Cvar_SetDescription( r_colorMipLevels, "Debugging tool to artificially color different mipmap levels so that they are more apparent (defaul 0)." );
+	ri.Cvar_SetDescription( r_colorMipLevels, "Debugging tool to artificially color different mipmap levels so that they are more apparent (default 0)." );
```
Comment (ec-) [inline code/renderervk/tr_init.c:1529]: no need to print default value in description

**Comment by KuehnhammerTobias** [comment]
Removed another useless 'default' value description (found by ec-).

**Pull request merged**